src/components/storage/HelpAutopartOptions.jsx

```diff
@@ -4,8 +4,8 @@ const _ = cockpit.gettext;
 
 export const helpEraseAll = _("Remove all partitions on the selected devices, including existing operating systems. Make sure you have backed up your data.");
 
-export const helpUseFreeSpace = _("Keep current disk layout and only install into available space.");
+export const helpUseFreeSpace = _("Keep current disk layout and use available space, to dual-boot with another OS.");
 
-export const helpMountPointMapping = _("Manually select storage locations for installation.");
+export const helpMountPointMapping = _("Select existing partitions as mount points. Useful for existing custom layouts.");
```

Comment:
Oh, that's based on an earlier version (up until this week :wink:). We were talking about revising it further in the chat, using select as the main title and assign as the sub-text (as to not be repetitive, as select is in the title and use is a different title for using the full disk).

I had "_Assign partitions to mount points. Intended for reusing an existing layout or after using a custom partitioner._" but I think we can make it shorter and still be fine enough... what do you think of something like this?

🔘 **Select existing partitions**
Assign partitions to mount points. Useful for pre-configured custom layouts.

(This removes the duplication of "select" and "existing" too.)

```suggestion
export const helpMountPointMapping = _("Assign partitions to mount points. Useful for pre-configured custom layouts.");
```

Reply:
(Also, I don't see the map mount points string changed in the diff, but perhaps I'm missing it?)